docs/integrations/builtin/credentials/hubspot.md

```diff
@@ -13,18 +13,14 @@ You can use these credentials to authenticate the following nodes:
 - [HubSpot](/integrations/builtin/app-nodes/n8n-nodes-base.hubspot/)
 - [HubSpot Trigger](/integrations/builtin/trigger-nodes/n8n-nodes-base.hubspottrigger/)
 
-## Prerequisites
-
-Create a [HubSpot](https://www.hubspot.com/){:target=_blank .external-link} account or [HubSpot developer](https://developers.hubspot.com/){:target=_blank .external-link} account.
-
 ## Supported authentication methods
 
-- App token: Used with [HubSpot](/integrations/builtin/app-nodes/n8n-nodes-base.hubspot/) node
-- Developer API key: Used with [HubSpot Trigger](/integrations/builtin/trigger-nodes/n8n-nodes-base.hubspottrigger/) node
-- OAuth2: Used with [HubSpot](/integrations/builtin/app-nodes/n8n-nodes-base.hubspot/) node
+- App token: Use with the [HubSpot](/integrations/builtin/app-nodes/n8n-nodes-base.hubspot/) node.
+- Developer API key: Use with the [HubSpot Trigger](/integrations/builtin/trigger-nodes/n8n-nodes-base.hubspottrigger/) node.
+- OAuth2: Use with the [HubSpot](/integrations/builtin/app-nodes/n8n-nodes-base.hubspot/) node.
 
 /// note | API key deprecated
```

Comment:
```suggestion
/// warning | API key deprecated
```

Reply:
@ayatnkw for deprecated things we usually just use a note--I used a warning for the X/Twitter thing since the behavior is wildly different between the two but everywhere else we use a note, which is why I went with it here. Though...in this case, since it's still in the UI, it probably does make sense to use warning since they won't get far if they use it. I'll commit your suggestion. :)